### PR TITLE
Remove the depth range hack option, turning it into an ugly game specific compatibility hack.

### PR DESCRIPTION
**Pull request by hrydgard**
I hate doing this, but it's not really better off as an option, as no other game needs it, as far as we know.

I don't think I've got all the game IDs out there, the list at gamefaqs has some holes - if you have a Phantasy Star with a different game ID let me know.

See #8171

### CODE REVIEW

File: assets/compat.ini [hunk 1]
```diff
@@ -78,3 +78,28 @@ PixelDepthRounding = true
...
+# Phantasy Star Portable has a strange depth clipping issue that we have been unable to solve other ways.
+[ULJM05309]
+DepthRangeHack = true
+[ULUS10410]
+DepthRangeHack = true
+[ULES01218]
+DepthRangeHack = true
+[ULJM08023]
+DepthRangeHack = true
+[ULES01218]
```
Comment (vnctdj) [inline assets/compat.ini]: Duplicate of line 88.

File: assets/compat.ini [hunk 2]
```diff
@@ -78,3 +78,28 @@ PixelDepthRounding = true
...
+DepthRangeHack = true
+
+# Phantasy Star Portable 2 has the same issue.
+[ULJM05493]
+DepthRangeHack = true
+[NPJH50043]
+DepthRangeHack = true
+[ULJM08030]
+DepthRangeHack = true
+[NPJH50043]
```
Comment (vnctdj) [inline assets/compat.ini]: Duplicate of line 98.

And "ULUS10529" is missing :)

**Comment by unknownbrackets** [comment]
Hmm.  See also #7069.  Was this also an issue in Phantasy Star Portable 1?  I thought it was only in Phantasy Star Portable 2 (#1788) as well as Phantasy Star Portable 2 Infinity which is its sequel.

I'm not sure we want to activate the hack for Phantasy Star Portable 1.

-[Unknown]

File: assets/compat.ini (hunk 2 above)
Comment (unknownbrackets) [inline assets/compat.ini]: Also missing:

ULJM91018 - Infinity demo disc?
NPJH90157 - Infinity demo
ULJM05732 - Infinity
NPJH50332 - Infinity

-[Unknown]

**Comment by hrydgard** [comment]
Hm, that may be right - I don't really know. I'll update it tonight.

**Comment by hrydgard** [comment]
Finally updated.

**Pull request merged**